mono/tests/Makefile.am

```diff
@@ -1538,7 +1538,22 @@ LLVM = $(filter --llvm, $(MONO_ENV_OPTIONS) $(AOT_BUILD_FLAGS))
 # Some of the tailcall tests pass native pointers to locals.
 #   Mono does not yet allow that.
 #   tailcall "i" is either calli, or integer instead of pointer, and the latter sidesteps the matter.
+#
+# https://github.com/mono/mono/issues/9617
+# flaky test delegate5 self_interrupt_thread: we can't resume from an async call
+# https://jenkins.mono-project.com/job/test-mono-pull-request-arm64-fullaot/1892/
+# Test Result (1 failure / +1)
+# MonoTests.runtime.delegate5.exe
+#
+# https://github.com/mono/mono/issues/9035
+# pinvoke3.exe flaky test
+# https://jenkins.mono-project.com/job/test-mono-pull-request-coop/5360/
+# Test Result (1 failure / +1)
+# MonoTests.runtime.pinvoke3.exe_timedout
+#
+# FIXME sort the lists
```

Comment:
these comments contain duplicated information and it is also hard to update/maintain. This would be enough:
```
delegate5.exe:  https://github.com/mono/mono/issues/9617
pinvoke3.exe: https://github.com/mono/mono/issues/9035
```

FIXME comment is a unrelated change.